cnd/migrations/2019-08-21-185335_create_swaps/up.sql

```diff
@@ -0,0 +1,10 @@
+-- MetadataStore only i.e., no StateStore.
+CREATE TABLE swaps (
+       swap_id TEXT NOT NULL PRIMARY KEY,
+       alpha_ledger VARCHAR NOT NULL,
+       beta_ledger VARCHAR NOT NULL,
+       alpha_asset VARCHAR NOT NULL,
+       beta_asset VARCHAR NOT NULL,
+       role VARCHAR NOT NULL,
+       counterparty INTEGER NOT NULL
```

Comment:
Integer?

Reply:
Cargo cult programming from the diesel docs.